src/StaffHub/Program.cs

```diff
@@ -9,7 +9,9 @@
     .AddInteractiveServerComponents();
 
 builder.Services.AddDbContext<ApplicationDbContext>(options =>
-    options.UseSqlServer(builder.Configuration.GetConnectionString("DefaultConnection")));
+    options.UseSqlServer(new ConfigurationBuilder().SetBasePath(Directory.GetCurrentDirectory())
+    .AddJsonFile("appsettings.Development.json", optional: true, reloadOnChange: true).Build()
+    .GetConnectionString("DefaultConnection")));
```

Comment:
We shouldn't even use `appsettings`. Instead of this use something like:
```
var connectionString = Environment.GetEnvironmentVariable("CONNECTION_STRING");
if (string.IsNullOrEmpty(connectionString)){
Console.WriteLine("NO CONNECTION STRING");
}
options.UseSqlServer(connectionString);
```